code/modules/antagonists/scamp/scamp_spawner.dm

```diff
@@ -0,0 +1,23 @@
+/*
+ * Scamps are minor antagonists so they can be used alongside other compatible gamemodes with ease.
+ * I really have no idea what i'm doing here.
+ */
+
+/datum/special_role/scamp
```

Comment:
okay forgive my ignorance but is this a datum we use??? I've never seen this before but it might have been added in the past year